d2networking/d2server/game_server.go

```diff
@@ -86,11 +87,11 @@ func NewGameServer(asset *d2asset.AssetManager, networkServer bool,
 		heroStateFactory:  heroStateFactory,
 	}
 
-	mapEngine := d2mapengine.CreateMapEngine(asset)
+	mapEngine := d2mapengine.CreateMapEngine(d2util.LogLevelDefault, asset) // need to be changed
 	mapEngine.SetSeed(gameServer.seed)
 	mapEngine.ResetMap(d2enum.RegionAct1Town, 100, 100)
 
-	mapGen, err := d2mapgen.NewMapGenerator(asset, mapEngine)
+	mapGen, err := d2mapgen.NewMapGenerator(asset, d2util.LogLevelDefault, mapEngine) // need to be changed
 	if err != nil {
 		return nil, err
 	}
```

Comment:
is there no way to get the log level from the app config ?

Reply:
I think, I'll implement logger in d2 networking in a future, so I'll change it